tests/Operation/WatchFunctionalTest.php

```diff
@@ -488,7 +488,7 @@ public function testRewindMultipleTimesWithNoResults(): void
     }
 
     #[DataProvider('provideCodecOptions')]
-    public function testNoChangeAfterResumeBeforeInsert(array $options): void
+    public function testNoChangeAfterResumeBeforeInsert(array $options, Closure $getIdentifier): void
```

Comment:
Let me know if there's some annotation or other comment you'd like to see used to denote that this parameter is unused.

Reply:
That could be `@phpstan-ignore constructor.unusedParameter`, but if PHPStan doesn't complain, that's fine not having it.